style.css

```diff
@@ -332,24 +350,28 @@ body {
   cursor: pointer;
   z-index: 1001;
 }
-.menu__btn > span,
-.menu__btn > span::before,
-.menu__btn > span::after {
+
+.menu__btn>span,
+.menu__btn>span::before,
+.menu__btn>span::after {
```

Comment:
What is this line doing?

Reply:
This part of code is responsible for menu button animation from three lines to x.